commerce-frontend-js/src/main/resources/META-INF/resources/js/info_box/_info_box.scss

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
no need for a blank line at the top I don't think

Reply:
Also same issues here with regard to selector sorting, rule sorting, and indentation